### DISCUSSION TITLE
Rewards for the top bug reporters February 2022

### DISCUSSION THREAD

**Discussion by Valsery** [chromiecraft/chromiecraft#3150] [General]
This is the number of VALID bug reports per every contributor that have been validated in February 2022.
When a bug is validated, it gets the [Linked [AC] label](https://github.com/chromiecraft/chromiecraft/issues?q=is%3Aissue+is%3Aopen+label%3A%22Linked+%5BAC%5D%22).

If your bug hasn't been validated yet, don't worry: it will be counted in a later month. Invalid bug reports will of course not be counted.

Source: https://www.chromiecraft.com/top-contributors-of-february-2022/

The people mentioned here: in order to receive your Chromie points, please create a **REDEEM REWARDS** ticket on our official Discord with the following:

- ping user `Gn#6416` (Valsery - Gn)
- your GitHub username
- your game account username
- your game account [email]

<img class="alignnone size-full wp-image-8570" src="https://www.chromiecraft.com/wp-content/uploads/2022/03/pinghere-1.png" alt="" width="929" height="490" />

Remember to give us a Star on our main GitHub repo: https://github.com/azerothcore/azerothcore-wotlk

<a href="https://github.com/FTGCocolandia">@FTGCocolandia</a> - 21 
<a href="https://github.com/Hyperbearr">@Hyperbearr</a> - 12
<a href="https://github.com/Luckdemon">@Luckdemon</a> - 8
<a href="https://github.com/Vladimirjov">@Vladimirjov</a> - 7
<a href="https://github.com/brookie2664">@brookie2664</a> - 4
<a href="https://github.com/Plasmashield">@Plasmashield</a> - 4
<a href="https://github.com/lellalex">@lellalex</a> - 3
<a href="https://github.com/Scrublettt">@Scrublettt</a> - 3
<a href="https://github.com/Josarth">@Josarth</a> - 2
<a href="https://github.com/vexima">@vexima</a> - 1
<a href="https://github.com/thom4466">@thom4466</a> - 1
<a href="https://github.com/silvidruid">@silvidruid</a> - 1
<a href="https://github.com/phonkala">@phonkala</a> - 1
<a href="https://github.com/oterfesk">@oterfesk</a> - 1
<a href="https://github.com/mandris36">@mandris36</a> - 1
<a href="https://github.com/lovelord83">@lovelord83</a> - 1
<a href="https://github.com/lNKU">@lNKU</a> - 1
<a href="https://github.com/kulkan1">@kulkan1</a> - 1
<a href="https://github.com/cachocarne">@cachocarne</a> - 1
<a href="https://github.com/baranzadeoglu">@baranzadeoglu</a> - 1
<a href="https://github.com/Zelfie">@Zelfie</a> - 1
<a href="https://github.com/WyldePointer">@WyldePointer</a> - 1
<a href="https://github.com/Tmetheorcfemale">@Tmetheorcfemale</a> - 1
<a href="https://github.com/Sathrenor">@Sathrenor</a> - 1
<a href="https://github.com/RobertCharlesPDX">@RobertCharlesPDX</a> - 1
<a href="https://github.com/OA350">@OA350</a> - 1
<a href="https://github.com/NoodlestheNoble">@NoodlestheNoble</a> - 1
<a href="https://github.com/MagnusHellstrom">@MagnusHellstrom</a> - 1
<a href="https://github.com/KaneoHunter">@KaneoHunter</a> - 1
<a href="https://github.com/Dejanos">@Dejanos</a> - 1
<a href="https://github.com/DarylBlake21">@DarylBlake21</a> - 1
<a href="https://github.com/BiroJenkins">@BiroJenkins</a> - 1
<a href="https://github.com/Arbit0">@Arbit0</a> - 1
<a href="https://github.com/Amber-Haze">@Amber-Haze</a> - 1

**Comment by FrancescoBorzi** [comment]
All people tagged here please read the above post

<a href="https://github.com/FTGCocolandia">@FTGCocolandia</a> - 21 
<a href="https://github.com/Hyperbearr">@Hyperbearr</a> - 12
<a href="https://github.com/Luckdemon">@Luckdemon</a> - 8
<a href="https://github.com/Vladimirjov">@Vladimirjov</a> - 7
<a href="https://github.com/brookie2664">@brookie2664</a> - 4
<a href="https://github.com/Plasmashield">@Plasmashield</a> - 4
<a href="https://github.com/lellalex">@lellalex</a> - 3
<a href="https://github.com/Scrublettt">@Scrublettt</a> - 3
<a href="https://github.com/Josarth">@Josarth</a> - 2
<a href="https://github.com/vexima">@vexima</a> - 1
<a href="https://github.com/thom4466">@thom4466</a> - 1
<a href="https://github.com/silvidruid">@silvidruid</a> - 1
<a href="https://github.com/phonkala">@phonkala</a> - 1
<a href="https://github.com/oterfesk">@oterfesk</a> - 1
<a href="https://github.com/mandris36">@mandris36</a> - 1
<a href="https://github.com/lovelord83">@lovelord83</a> - 1
<a href="https://github.com/lNKU">@lNKU</a> - 1
<a href="https://github.com/kulkan1">@kulkan1</a> - 1
<a href="https://github.com/cachocarne">@cachocarne</a> - 1
<a href="https://github.com/baranzadeoglu">@baranzadeoglu</a> - 1
<a href="https://github.com/Zelfie">@Zelfie</a> - 1
<a href="https://github.com/WyldePointer">@WyldePointer</a> - 1
<a href="https://github.com/Tmetheorcfemale">@Tmetheorcfemale</a> - 1
<a href="https://github.com/Sathrenor">@Sathrenor</a> - 1
<a href="https://github.com/RobertCharlesPDX">@RobertCharlesPDX</a> - 1
<a href="https://github.com/OA350">@OA350</a> - 1
<a href="https://github.com/NoodlestheNoble">@NoodlestheNoble</a> - 1
<a href="https://github.com/MagnusHellstrom">@MagnusHellstrom</a> - 1
<a href="https://github.com/KaneoHunter">@KaneoHunter</a> - 1
<a href="https://github.com/Dejanos">@Dejanos</a> - 1
<a href="https://github.com/DarylBlake21">@DarylBlake21</a> - 1
<a href="https://github.com/BiroJenkins">@BiroJenkins</a> - 1
<a href="https://github.com/Arbit0">@Arbit0</a> - 1
<a href="https://github.com/Amber-Haze">@Amber-Haze</a> - 1
